src/components/EducationContainer/EducationContainer.jsx

```diff
@@ -3,9 +3,19 @@ import "./EducationContainer.css";
 
 export const EducationContainer = () => {
   const [education, setEducation] = useState([]);
-
+  // Create a function to sanitize HTML content using the global sanitizeHTML function
+  const sanitizeHTML = (html) => {
+    return window.sanitizeHTML(html);
+  };
```

Comment:
This same definition is reused across multiple files. Would it be beneficial to define it once and import it everywhere else?

Reply:
@fissoreg Resolved